File: FrozenApp/src/main/java/com/github/lakeshire/frozen/MovieService.java
package com.github.lakeshire.frozen;

import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

/**
 * Created by liuhan on 17/9/1.
 */

public interface MovieService {

    @GET("top250")
    Observable<ListEntity<Movie>> getTopMovie(@Query("start") int start, @Query("count") int count);
}

File: FrozenApp/src/main/java/com/github/lakeshire/frozen/manager/http/MovieService.kt
package com.github.lakeshire.frozen.manager.http

import com.github.lakeshire.frozen.model.ListEntity
import com.github.lakeshire.frozen.model.Movie
import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

interface MovieService {

    @GET("top250")
    fun getTopMovie(@Query("start") start: Int, @Query("count") count: Int): Observable<ListEntity<Movie>>
}
